src/Xamarin.Android.Build.Tasks/Tasks/Aapt2Link.cs

```diff
@@ -263,6 +265,16 @@ public async override System.Threading.Tasks.Task RunTaskAsync ()
 				}
 			}
 
+			if (AdditionalAndroidAssetPaths != null) {
+				for (int i = AdditionalAndroidAssetPaths.Length - 1; i >= 0; i--) {
+					var assetDir = AdditionalAndroidAssetPaths [i].ItemSpec.TrimEnd ('\\');
+					if (!string.IsNullOrWhiteSpace (assetDir) && Directory.Exists (assetDir)) {
+						cmd.Add ("-A");
+						cmd.Add (GetFullPath (assetDir));
+					}
+				}
+			}
```

Comment:
Would `<ResolveLibraryProjectImports/>` also need to skip extracting `assets` directories?

Reply:
In this case, we are still extracting the directories to the `lp` directory, but we skip the copying over to the global assets directory and use them in place. This is a first step to allow us to support using assets/res/etc from the project reference/nuget location.

Reply:
We already had a concept of `AdditionalResourcePaths` but did't have the same for assets. I figured this would be an easy win, especially if you have large assets (like movie files).